snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml

```diff
@@ -80,6 +80,10 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.13.1.5.1.4
           name: ciscoEnvMonSupplySource
         tag: power_source
+      - column:
+          OID: 1.3.6.1.4.1.9.9.13.1.5.1.2
+          name: ciscoEnvMonSupplyStatusDescr
+        tag: power_status_descr
```

Comment:
Usually using a description as tag is not great, but it seems we don't have much choice, the only fields available for this table are:

```
CiscoEnvMonSupplyStatusEntry ::=
        SEQUENCE {
                ciscoEnvMonSupplyStatusIndex    Integer32,
                ciscoEnvMonSupplyStatusDescr    DisplayString,
                ciscoEnvMonSupplyState          CiscoEnvMonState,
                ciscoEnvMonSupplySource         INTEGER
        }
```

I though about `ciscoEnvMonSupplyStatusIndex` but it not great either since it has no meaning:

> Unique index for the power supply being instrumented.
>                This index is for SNMP purposes only, and has no
>                intrinsic meaning.

Reply:
I also considered using "ciscoEnvMonSupplyStatusIndex"... But if the end-user gets the internal ID, with no meaning, that would be barely usable...